lib/main_common.pm

```diff
@@ -472,7 +471,7 @@ sub load_extra_tests {
         if (check_var('DISTRI', 'sle')) {
             # start extra x11 tests from here
             loadtest 'x11/vnc_two_passwords';
-            # TODO: check why this is not called on opensuse
+            # because of bsc#978027, it is not called for opensuse
```

Comment:
this bug is very unlikely to be fixed anytime soon. We should not exclude the module because of that.